examples/03_datetime_encoder.py

```diff
@@ -86,7 +83,7 @@
 )
 
 X_enc = encoder.fit_transform(X)
-encoder.get_feature_names_out()
+print(encoder.get_feature_names_out())
```

Comment:
Without pprint:
<img width="990" alt="image" src="https://github.com/skrub-data/skrub/assets/65392801/00012537-b999-4b54-8927-dbd895fe2c0b">
With print instead of pprint:
<img width="802" alt="image" src="https://github.com/skrub-data/skrub/assets/65392801/57b9aa25-8271-4294-bae5-9e9d41f97e60">


I think second option is better because all text can be seen on the screen

Reply:
WDYT?

Reply:
Ok, this makes sense. Let's replace `pprint` with `print` then.

Reply:
After reviewing the notebook, `print(*..., sep="\n")` feels too complex. This is not what users will use.
Let's keep the `pprint` instead and add a `pprint` where you initially added a `print`.

Reply:
Done!